osgtest/tests/special_install.py

```diff
@@ -30,10 +30,15 @@ def test_02_install_packages(self):
         fail_msg = ''
         pkg_repo_dict = OrderedDict((x, core.options.extrarepos) for x in core.options.packages)
 
-        # HACK: Install slurm out of development if we're running 'All' tests.
+        # HACK: Install Slurm and osg-tested-internal out of development-like repos.
         # SOFTWARE-1733 may one day give us a generalized solution.
+        if float(core.osg_release()) > 3.4:
+            devops_repo = 'devops-itb'
```

Comment:
Not critical but converting to `float` first feels a bit nasty.  (And won't work for 3.10+)

If you wanna leave it as is, fine, otherwise `PackageVersion('osg-release') > '3.4'` does a proper rpm version ordering.

Reply:
Yeah, it felt dirty. I'll fix it.

Reply:
I decided against the direct `osg-release` query since `core.osg_release()` gives us nice handling of `osg-release` vs `osg-release-itb`